### PR TITLE
Add support for Dokku 0.4.x+

### PR DESCRIPTION
**Pull request by josegonzalez**

### CODE REVIEW

**Comment by josegonzalez** [comment]
I'm definitely not a big fan of modifying the dokku-core. I'll take a look and see how we might expose that data elsewhere. We might be able to get away with writing a temporary file and reading it out in post-deploy.

**Comment by nickstenning** [comment]
:+1:

**Comment by nickstenning** [comment]
Sorry for the delay. I've added you as a contributor here.

File: README.md [hunk 1]
```diff
@@ -6,8 +6,21 @@ to build/deployment stages.
 
 ## Installation
 
-    git clone https://github.com/nickstenning/dokku-webhooks.git /var/lib/dokku/plugins/webhooks
-    mv /var/lib/dokku/plugins/git/receive-app /var/lib/dokku/plugins/git/receive-app.bak
+
+```shell
+# dokku 0.4.x+
+sudo dokku plugin:install https://github.com/dokku/dokku-letsencrypt.git
```
Comment (jezdez) [inline README.md:12]: oops, typo here

Comment (nickstenning) [reply README.md:12]: Gah! Thanks Jannis: https://github.com/nickstenning/dokku-webhooks/commit/8aa1c3c2cacff50eff49510de7e77e806c67f9a9

**Pull request merged**